Source/info.c

```diff
@@ -7783,6 +7783,60 @@ static const char cr_yellow[]=
      224,225,226,227,228,229,230,231,232,233,234,235,236,237,238,239,
      240,241,242,243,244,245,246,247,248,249,250,251,252,253,254,255};
 
+static const char cr_black[] =
```

Comment:
I have revised cr_black and cr_white at least once since the initial commit. Are these now the latest revisions?

Reply:
Yes, I recheck with the latest master of pr+.